src/transformers/models/speech_to_text/modeling_tf_speech_to_text.py

```diff
@@ -942,22 +942,6 @@ def get_embed_tokens(self):
     def set_embed_tokens(self, embed_tokens):
         self.embed_tokens = embed_tokens
 
-    def _prepare_decoder_attention_mask(self, attention_mask, input_shape, inputs_embeds, past_key_values_length):
```

Comment:
This function causes issues in graph mode, as `combined_attention_mask` can be either a tensor or None. Instead, the equivalent logic in bart call method is used in this model's to safely create `combined_attention_mask`